content/blog/2021/12/2021-12-03-election-results.adoc

```diff
@@ -0,0 +1,100 @@
+If you are interested to learn more, please see the blog post below.
+
+== Governance Board election details
+
+This year we had link:https://community.jenkins.io/t/jenkins-election-candidates/776#board-members-2[three candidates] participating in Jenkins Governance Board elections:
+Mark Waite,  Oleg Nenashev (returning), and Ullrich Hafner (returning)
+All of them are awesome community leaders who actively contribute to the Jenkins project and represent its users.
+It would be an honor to have them on the Jenkins board.
+Regardless of the election results, we appreciate their participation and the time they invested in these elections.
+
+This year we were electing 2 governance board members.
+We were using the https://civs.cs.cornell.edu/[Condorcet Internet Voting Service] that allows voters to rank their choices rather than just picking their one favorite choice.
+You can find full voting results link:https://civs.cs.cornell.edu/cgi-bin/results.pl?id=E_c206f1625b47c65d[here]:
+
+1. **Mark Waite** (Condorcet winner: wins contests with all other choices)
+2. **Oleg Nenashev** loses to Mark Waite by 37–22
+3. Ullrich Hafner loses to Mark Waite by 42–14, loses to Oleg Nenashev by 37–15
+
+Congratulations to the returning Oleg, and the newly elected Mark, and thanks to Ullrich for his past contributions.
+All new board members are elected for a 2-year term unless they decide to step down earlier.
+The estimated end of the term for them is December 02, 2022.
+
+== Officer election details
+
+All 5 officer positions were up for election this year.
+These roles have a 1-year term, with the estimated end of term on Dec 02, 2022.
+After the initial review of nominations and confirmations with potential candidates,
+4 officer positions were uncontested:
+
+* **Tim Jacomb** - link:/project/team-leads/#release[Release Officer]
+* **Alyssa Tong** - link:/project/team-leads/#events[Events Officer]
+* **Damien Duportal** - link:/project/team-leads/#infrastructure[Infrastructure Officer]
+* **Wadeck Follonier** - link:/project/team-leads/#security[Security Officer]
+
+Documentation Officer election results::
+link:https://github.com/MarkEWaite[Mark Waite] won the biggest support as a Documentation officer (link:https://civs.cs.cornell.edu/cgi-bin/results.pl?id=E_df9bdfde927d6f2f[voting results]).
+Mark will continue as link:/project/team-leads/#documentation[Documentation Officer].
+
+1. **Mark Waite** (Condorcet winner: wins contests with all other choices)
+2. Saeid Bostandoust loses to Mark Waite by 46–9
+
+Thanks to all Jenkins officers for their continued leadership!
+Officers take responsibility for many day-to-day processes in the Jenkins community and lead the contributor teams working on them.
+It requires significant time commitment, and it is not taken for granted.
+
+=== Statistics
+
+This year we had 81 registered voters and around 59 actual votes.
+
+== Feedback
+
+TODO - this section
+
+Jenkins project plans to conduct elections every year.
```

Comment:
I don't know what we should put here.

@olblak are you running the retro?

Reply:
I created a retro document as a copy of 2020 with minor changes for 2021.